unittests/InstructionCountCI/FEXOpt/MultiInst.json

```diff
@@ -225,6 +225,25 @@
         "mov x0, x5",
         "mov x1, x11",
         "mov x2, x10",
+        "cbz x0, #+0x60",
+        "orr x3, x1, x2",
+        "and x3, x3, #0x3",
+        "cbnz x3, #+0x44",
+        "sub x0, x0, #0x40 (64)",
+        "tbnz x0, #63, #+0x18",
+        "ldp q0, q1, [x2], #32",
+        "stp q0, q1, [x1], #32",
+        "ldp q0, q1, [x2], #32",
+        "stp q0, q1, [x1], #32",
+        "b #-0x18",
```

Comment:
It's typically profitable to [invert loops](https://en.wikipedia.org/wiki/Loop_inversion). Instead of (sub/tbnz/ldp/stp/ldp/stp/b), ideally it'd be sub/tbnz up front and then a (ldp/stp/ldp/stp/sub/tbz) loop. Overall, this saves 2 instructions per iteration. Ditto for the memset loops.